### PR TITLE
Readme 1

### PR DESCRIPTION
**Pull request by ralphtheninja**
I wanted to make this quick brush over first, before rewriting different sections. Also, I think we are using bold too much.

### CODE REVIEW

**Comment by ralphtheninja** [comment]
This is up for discussion. I can always adjust this PR, rebase etc. #465 should be merged first.

File: CHANGELOG.md.~HEAD^~ [hunk 1]
```diff
@@ -0,0 +1,246 @@
+### 1.0.0 @ May 14 2015
```
Comment (vweevers) [inline CHANGELOG.md.~HEAD^~]: What's this file?

Comment (ralphtheninja) [reply CHANGELOG.md.~HEAD^~]: Aah damn. Sorry it's junk. Removing.

Comment (ralphtheninja) [reply CHANGELOG.md.~HEAD^~]: Done. Accidentally commited that file when rewriting the other PR (use `vc-revision-other-window` command in emacs to check out previous versions of a file)

**Comment by vweevers** [comment]
> I wanted to make this quick brush over first

> #465 should be merged first.

I'm confused. Which one goes first and needs review?

**Comment by ralphtheninja** [comment]
@vweevers Sorry for the confusion. Entirely my fault since I'm impatient :) There are two PRs and this is based on top of #465 , which needs review and merge before this is merged.

**Comment by ralphtheninja** [comment]
Closing this. I'm just messing things up anyway :)

**Pull request closed without merging**